futurenhs.automation/features/search.feature

```diff
@@ -47,14 +46,13 @@ Feature: Search
 
 
     Scenario: FNHS100 - Search for a Group That Doesn't Exist
-        When I search for 'Automation Group'
+        When I search for 'Automaton Group'
```

Comment:
This is spelt wrong you sure this won't effect your run in correlation to the data?

Reply:
Yes that's intentional, as the group _shouldn't_ exist in this search.